07-sampling.Rmd

```diff
@@ -1064,33 +1062,245 @@ Comment on the representativeness of the following *sampling methodologies*:
 
-Shuyi Chiou, Casey Dunn, and Pathikrit Bhattacharyya created a 3-minute and 38-second video at <https://youtu.be/jvoxEYmQHNM> explaining this crucial statistical theorem using the average weight of wild bunny rabbits and the average wingspan of dragons as examples. Figure \@ref(fig:CLT-video-preview) shows a preview of this video.
 
-```{r CLT-video-preview, echo=FALSE, fig.cap="Preview of Central Limit Theorem video.", purl=FALSE, out.width = "75%"}
-knitr::include_graphics("images/copyright/CLT_video_preview.png")
+### The distribution of $\bar X$
+
+The sampling distribution of the sample proportion (or sample mean) is now represented by the distribution of $\bar X$. Think of this distribution as how often each of the values that $\bar X$ could take may end up happening. 
+
+A very useful way to learn about this distribution is using simulations. 
+Simulations seldom provide the exact structure of the distribution, because we would need an infinite number of samples, but a large number of replications often produce a really good approximation of the distribution and we can use it to study the distribution's characteristics. We now repeat our experiment 10,000 times. 
+
+```{r}
+simul_X_bar_25 <- bowl |>
+  rep_sample_n(size = 25, replace = TRUE, reps = 10000) |>
+  summarize(prop_red = mean(color == "red"))
 ```
 
-Here's what is so surprising about the Central Limit Theorem: regardless of the shape of the underlying population distribution, the sampling distribution of means (such as the sample mean of bunny weights or the sample mean of the length of dragon wings) and proportions (such as the sample proportion red in our shovels) will be **normal.** Normal distributions are defined by where they are centered and how wide they are, and the Central Limit Theorem gives us both:
+The object `simul_X_bar` contains 10,000 sample proportions, each replicate was obtained following the experiment described earlier, so each sample proportion is a possible value of $\bar X$. 
 
-1. The sampling distribution of the point estimate is centered at the true population parameter 
-2. We have an estimate for how wide the sampling distribution of the point estimate is, given by the standard error (which we will discuss further in Chapter \@ref(confidence-intervals))
+### The mean and standard deviation of $\bar X$
+
+Based on probability theory, the mean of $\bar X$ should be equal to the population proportion of red balls in the bowl, that is $p = 900/2400 = 0.375$, and the standard deviation of $\bar X$, also called the standard error, is given by the formula: $$SE_{\bar X} = \sqrt{\frac{p(1-p)}{n}} = \sqrt{\frac{0.375\cdot(1-0.375)}{25}} = 0.0968$$
```

Comment:
I think a reference back to what we've seen previously (before the theory) would be helpful here and then formalize it.